pkg/virtctl/vnc/vnc.go

```diff
@@ -49,6 +51,7 @@ func NewCommand(clientConfig clientcmd.ClientConfig) *cobra.Command {
 		},
 	}
 	cmd.SetUsageTemplate(templates.UsageTemplate())
+	cmd.Flags().BoolVarP(&debug, "debug", "D", false, "enable debug output")
```

Comment:
Instead of introducing that flag, could you reuse the `-v`/`--verbose` flag? Let's say append the `--debug` flag to remote-viewer if we have a verbosity level higher or equal to 4.